backend/sites/src/app/entities/siteDocPartics.entity.ts

```diff
@@ -24,15 +24,23 @@ export class SiteDocPartics extends ChangeAuditEntity {
   @Column('bigint', { primary: true, name: 'id' })
   id: string;
 
+  // @Field()
+  // @Column('character varying', { name: 'dpr_code', unique: true, length: 6 })
+  // dprCode: string;
+
+  //Need to remove unique constraint because as per stand-up discussion everyone
+  // will be ATH i.e Author. I would suggest to keep this null able as we don't have
+  // UI for dpr code.
```

Comment:
same for this, as dprcode was unique and a composite key, I make nullable because we don't need it as per current application UI. However keeping it commented, in case there will be sync issue, once everything get tested will remove the commented  code